### PR TITLE
docs(generate): add nojekyll option in generator

### PR DESCRIPTION
**Pull request by amotarao**
Add to generator.nojekyll options about https://github.com/nuxt/nuxt.js/pull/8813

### CODE REVIEW

**Review by clarkdo — DISMISSED** [review]

File: content/en/_archives/api/configuration-generate.md [hunk 1]
```diff
@@ -197,6 +197,13 @@ Interval (in milliseconds) between two render cycles to avoid flooding a potenti
 - **Deprecated!**
 - Use [build.html.minify](/api/configuration-build#html-minify) instead
 
+## nojekyll
+
+- Type: `Boolean`
+- Default: `true`
+
+By default, adding `.nojekyll` file for serving static pages on GitHub Pages. If you want to disable this feature you can set the value to `false`
```
Comment (danielroe) [inline content/en/_archives/api/configuration-generate.md:205]:
```suggestion
Adds a `.nojekyll` file to enable serving `/_nuxt/` when deployed on GitHub Pages. If you want to disable this feature you can set the value to `false`.
```

Comment (amotarao) [reply content/en/_archives/api/configuration-generate.md]: Thank you for your suggestion. I committed it.

**Comment by danielroe** [comment]
Would you also change the active documentation, not just the archives?

**Comment by amotarao** [comment]
@danielroe I changed it. Do I have to delete from archives?

**Comment by amotarao** [comment]
Thank you @danielroe , I fixed it.

**Review by danielroe — DISMISSED** [review]

File: content/en/guides/configuration-glossary/configuration-generate.md [hunk 2]
```diff
@@ -193,6 +193,13 @@ Interval in milliseconds between two render cycles to avoid flooding a potential
 - **Deprecated!**
 - Use [build.html.minify](/docs/2.x/configuration-glossary/configuration-build#htmlminify) instead
 
+## nojekyll
+
+- Type: `Boolean`
+- Default: `true`
+
+By default, adding `.nojekyll` file for serving static pages on GitHub Pages. If you want to disable this feature you can set the value to `false`
```
Comment (danielroe) [inline content/en/guides/configuration-glossary/configuration-generate.md:201]:
```suggestion
Adds a `.nojekyll` file to enable serving `/_nuxt/` when deployed on GitHub Pages. If you want to disable this feature you can set the value to `false`.
```

**Comment by danielroe** [comment]
Note: pending nuxt/nuxt.js#8813

**Comment by pi0** [comment]
@Atinux This feature is not landed yet :)

**Comment by danielroe** [comment]
I think it's a great way to drive early adoption 😆

**Pull request merged**